setup.py

```diff
@@ -22,7 +22,7 @@
 install_reqs = [
     'appdirs', 'colorama>=0.3.3', 'jinja2',
     'sh>=1.10, <2.0; sys_platform!="win32"',
-    'build', 'toml', 'packaging', 'setuptools'
+    'build', 'toml', 'packaging', 'setuptools', 'wheel'
```

Comment:
I'm slowly trying to enforce version pinning for deps anywhere, to avoid potential headaches, can we pin it with `~=` ?

(Renovatebot will send out notifications and open PRs for outdated deps)